docs/overview.rst

```diff
@@ -0,0 +1,56 @@
+PIE Service Overview
+--------------------
+The program intent and engagement (PIE) service is a data store used to capture
+various interactions a user could have with a program and its courses.
+
+Users of the service can query it for a user's intents, or create new intents
+via HTTP request or management command.
+
+Program Intent Model
+~~~~~~~~~~~~~~~~~~~~
+The program intent model is meant to represent when a learner marked an interest in a program, why we believe
+their action represents an intent, and how certain we are of their intent to participate in a program.
+
+The types of intent reasons and their corresponding certainty are described in the following table
+
++----------------------+--------------------------------------------+-------------+
+| Reason               | Description                                | Certainty   |
++======================+============================================+=============+
+| BUNDLE_PURCHASE      | User purchased program as a bundle         | CERTAIN_YES |
++----------------------+--------------------------------------------+-------------+
+| ENROLLED_ALL_COURSES | User has enrolled in all program courses   | MAYBE       |
++----------------------+--------------------------------------------+-------------+
+| EARNED_PROGRAM_CERT  | User has earned a program certificate      | CERTAIN_YES |
++----------------------+--------------------------------------------+-------------+
+
+Accessing Learner Intents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Learner intents can be queried via HTTP GET request. The request user will be used to filter
+intents, so only intents associated with that user will be returned.
+
+Currently the implementation will only return the most recent and most certain intent
+per program for each user.
+
+Inserting Learner Intents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+There are two methods to create new learner intents:
+
+- HTTP POST request
+- Insert management command
+
+A learner intent can be created via HTTP POST request. The API expects the request data to contain::
```

Comment:
we should say that this creates intents for the current user and only the current user